lib/aullar/styling.moon

```diff
@@ -4,6 +4,7 @@
 {:fill} = require 'ffi'
 {:define_class} = require 'aullar.util'
 GapBuffer = require 'aullar.gap_buffer'
+Markers = require 'aullar.markers'
```

Comment:
@nilnor Ah, right, I wanted to ask: is there a reason for `set` to be public? I couldn't find any uses outside of `apply`, and that adds the markers itself more efficiently (it collects them all together and batch-updates them). Could `set` be made into `_set`?